Packs/CommonTypes/Layouts/layoutscontainer-unifiedFile.json

```diff
@@ -158,7 +158,7 @@
                             },
                             {
                                 "endCol": 2,
-                                "fieldId": "signatureoriginalname",
+                                "fieldId": "signature",
```

Comment:
On some cases you changed one fieldId to another - why?

Reply:
After consulting with @yaakovi we decided that in cases where there are incident fields that belong to the pack and it does not appear in the layout, but there is a field in the layout that is similar to it (and did not exist in the content), Apparently this is a mistake in the name of the field and the author's reference is to the field that exists in the pack.